apps/benchmarks/benchmarks.nim

```diff
@@ -30,7 +30,7 @@ proc main(): Future[string] {.async, gcsafe.} =
   var proofVerTimes: seq[times.Duration] = @[]
   for i in 0 .. 50:
     var time = getTime()
-    let proof = manager.generateProof(data, getCurrentEpoch()).get()
+    let proof = manager.generateProof(data, default(Epoch)).get()
```

Comment:
this will default to 0, is that okay?

Reply:
yep all good. its just to benchmark proof generation, so the epoch here is not really important